x/ibc/02-client/types/expected_keepers.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+// StakingKeeper expected staking keeper
+type StakingKeeper interface {
+	GetHistoricalInfo(ctx sdk.Context, height int64) (stakingtypes.HistoricalInfo, bool)
```

Comment:
One way I resolved this import cycle was by wrapping `staking.HistoricalInfo` in an interface here that had `GetValSet` and `GetHeader`. Might be cleaner, but this works.